core/session.ts

```diff
@@ -193,6 +195,14 @@ export class Session {
     }
   }
 
+  public notebookAction(notebookConfig: dataform.ActionConfig, notebookContents: string): Notebook {
```

Comment:
this is part of our public API, and should follow that consistently, i.e. `notebook()`